webdriver/tests/bidi/browsing_context/__init__.py

```diff
@@ -34,15 +41,22 @@ def assert_browsing_context(
     assert info["url"] == url
 
 
-def assert_navigation_info(event, context, url):
-    assert "context" in event
-    assert isinstance(event["context"], str)
-    assert event["context"] == context
+def assert_navigation_info(event, expected_navigation_info):
+    recursive_compare(
+        {
+            "context": any_string,
+            "navigation": any_string_or_null,
+            "timestamp": any_int,
+            "url": any_string,
+        },
+        event,
+    )
+
+    if "context" in expected_navigation_info:
+        assert event["context"] == expected_navigation_info["context"]
 
-    assert "url" in event
-    assert isinstance(event["url"], str)
-    assert event["url"] == url
+    if "navigation" in expected_navigation_info:
+        assert event["navigation"] == expected_navigation_info["navigation"]
```

Comment:
looks like this assertion is not used anymore

Reply:
It's actually not used yet given that at least on our side we do not have a navigation id to check. As such all of the tests would be broken. Do your implementation send an id? If yes maybe we can make some assertions browser dependent for now?